contracts/smart-account/SmartAccount.sol

```diff
@@ -222,12 +222,12 @@ contract SmartAccount is
      * @param value Amounts of native tokens to send along with the transactions
      * @param func Data of the transactions
      */
-    function executeBatchCall(
+    function executeBatch(
```

Comment:
Update the comments of this function to include new function name `executeBatch_y6U`